core/persist/viewmanager/Types.ts

```diff
@@ -1,35 +1,39 @@
-import {PlainObject} from '@xh/hoist/core';
-
 /**
- * Interface for a bundle of {@link PersistableState} managed by {@link ViewManagerModel}.
- * Views are persisted to / loaded from the server as {@link JsonBlob}s.
+ * Interface for metadata describing a bundle of {@link PersistableState} managed by
+ * {@link ViewManagerModel}. Views are persisted to / loaded from the server as {@link JsonBlob}s.
  */
-export interface View<T extends PlainObject = PlainObject> {
-    /** Either null for private views or special token "*" for globally shared views. */
-    acl: '*' | null;
-    dateCreated: number;
+export interface ViewInfo {
```

Comment:
I'm surprised that I can't think of another term to use here (vs `Info`) - casting about for something that we've already used in an analogous context.  I am sure you guys discussed at length...  

I don't think our go-to `Spec` applies, since this is not a "recipe to make one".  Did you consider `Meta` or `Metadata`?  

Not at all critical, happy to proceed with this, but one of those cases where I feel like I can't quite put my finger on the word that captures the concept.

Reply:
Agreed - we spent a good amount of time trying to come up with the right name, and not sure we quite landed it.  Originally, I had `View`, `State` and `ViewWithState`.  That was a little wonky.  I had the same though re: spec.  I thought of `Meta` or `ViewMeta` and still think that fits reasonably well.